simplicity-sys/Cargo.toml

```diff
@@ -10,6 +10,7 @@ cc = "1.0"
 
 [dependencies]
 libc = "0.2"
+bitcoin_hashes = "0.11"
```

Comment:
No need to use seperate dependancy. I think we can get it from `elements::bitcoin::hashes`

Reply:
`simplicity-sys` is a separate crate with separate dependencies. Importantly, the crate cannot use (rust-)`simplicity` or its dependencies. I can add an `elements` dependency, but it this better than `bitcoin_hashes`, which is already a dependency of `simplicity`?

Reply:
We have to keep `simplicity` and `simplicity-sys` in sync, dependency-wise, but I think we can manage.

Reply:
Yep, this is needed. Sorry about that.